jboss/container/util/pkg-update/execute.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+
+for candidate in yum dnf microdnf; do
+    if command -v "$candidate"; then
+        mgr="$(command -v "$candidate")"
+        "$mgr" update -y
+        "$mgr" -y clean all
```

Comment:
Should we combine these lines? The intention for this is to clean files which got cached due to the `$mgr update -y` command, right? I.e. `"$mgr" update -y && "$mgr" -y clean all`

Reply:
The whole script gets executed in one container layer so there's no need to do that

Reply:
OK.